revolv/templates/administrator/dashboard.html

```diff
@@ -3,143 +3,151 @@
+              <img class="project-img" src='{{project.cover_photo.url}}'/>
+              <div class="overlay"></div>
+              <div class="status-bar">
+                <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
+              </div>
+            </div>
+          {% endfor %}
+          {% for project in completed_projects %}
+            <div class="project-container large-4 columns">
+              <img class="project-img" src='{{project.cover_photo.url}}'/>
+              <div class="overlay"></div>
+              <div class="status-bar">
+                <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
+              </div>
+            </div>
+          {% endfor %}
+         </div>
+       </div>
+         <div class="large-3 pull-9 columns>">
+          <br></br>
+          <div class="large-10 pull--1 columns>">
+          <ul class="accordion" data-accordion>
+            <li class="accordion-navigation active">
+              <a href="{% url "project:new" %}">Create New Project</a>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel1">Active Projects</a>
+              <div id="panel1" class="content">
+                <ul class="side-nav">
+                  {% for project in active_projects %}
+                    <li><a href="{% url 'project:view' project.id %}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel2">Proposed Projects</a>
+              <div id="panel2" class="content">
+                <ul class="side-nav">
+                  {% for project in proposed_projects %}
+                    <li><a href="{% url 'project:view' project.id %}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel3">Drafted Projects</a>
+              <div id="panel3" class="content">
+                <ul class="side-nav">
+                  {% for project in drafted_projects %}
+                    <li><a href="{% url 'project:view' project.id %}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel4">Completed Projects</a>
+              <div id="panel4" class="content">
+                <ul class="side-nav">
+                  {% for project in completed_projects %}
+                    <li><a href="{% url 'project:view' project.id %}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+          </ul>
+         </div>
         </div>
+      </div>
     </div>
-</div>
-
+        <br></br>      
```

Comment:
Sorry that was a little bit fanatic haha. br just shouldn't be used for spacing between elements, that's what margin is for.